src/libpgmoneta/prometheus.c

```diff
@@ -949,7 +952,11 @@ general_information(int client_fd)
    data = pgmoneta_append(data, "pgmoneta_state ");
    data = pgmoneta_append(data, "1");
    data = pgmoneta_append(data, "\n\n");
-
+   data = pgmoneta_append(data, "#HELP pgmoneta_version The version of pgmoneta\n");
+   data = pgmoneta_append(data, "#TYPE pgmoneta_version gauge\n");
+   data = pgmoneta_append(data, "pgmoneta_version ");
```

Comment:
You have to do this as

```
pgmoneta_version{version="0.11.0"} 1
```

Check it in Prometheus / Grafana

Reply:
what is 1 supposed to mean?
and am I not supposed to add this #TYPE code 
or the `pgmoneta_version{version="0.11.0"} 1` will be written here only?


Sorry for late reply. I had my farewell going on in my college. So could not get back.

Reply:
We need the `1` because that is how Prometheus works